packages/core/src/fhir-to-cda/cda-templates/commons.ts

```diff
@@ -1,29 +1,43 @@
 import {
   Address,
   CodeableConcept,
+  Coding,
   ContactPoint,
   Identifier,
   Organization,
 } from "@medplum/fhirtypes";
+import { normalizeOid } from "@metriport/shared";
+import {
+  assigningAuthorityNameAttribute,
+  extensionAttribute,
+  inlineTextAttribute,
+  nullFlavorAttribute,
+  rootAttribute,
+  useAttribute,
+  valueAttribute,
+  xmlnsXsiAttribute,
+  xsiTypeAttribute,
+} from "./constants";
 import {
   CDAAddress,
   CDACodeCE,
   CDACodeCV,
   CDAInstanceIdentifier,
   CDAOrganization,
   CDATelecom,
+  CDAValueST,
   Entry,
   EntryObject,
 } from "../cda-types/shared-types";
-import {
-  assigningAuthorityNameAttribute,
-  extensionAttribute,
-  nullFlavorAttribute,
-  rootAttribute,
-  useAttribute,
-  valueAttribute,
-} from "./constants";
 
+const CODING_MAP = new Map<string, string>();
+CODING_MAP.set("http://loinc.org", "2.16.840.1.113883.6.1");
```

Comment:
for these, could we use the shared constants here?
https://github.com/metriport/metriport/blob/031d29a3ce1abcd330c9bfb8dca0902dc8c71f23/packages/core/src/external/carequality/shared.ts#L15